dist/angular-spring-data-rest.js

```diff
@@ -261,12 +267,13 @@ angular.module("spring-data-rest").provider("SpringDataRestAdapter", function ()
                                     // 1. the all link names key is given then fetch the link
```

Comment:
I would change the order of the comments here and add the following one as fist comment:

```
// 1. the link was not fetched already.
```